### PR TITLE
Add Lua module file

### PR DESCRIPTION
**Pull request by nahceigna**
Update [Software installation page ](https://docs.hpc.shef.ac.uk/en/latest/hpc/installing-software.html#making-software-available-via-a-custom-module-file) to encourage users to use Lua on Stanage.

### CODE REVIEW

File: referenceinfo/environment-modules/creating-custom-modulefiles.rst [hunk 1]
```diff
@@ -13,74 +13,153 @@ module files you can add a directory called modules to your home directory
...
+
+            Module files are **not aware of bash shell variables unless you import them** from the env array 
+            and set a TCL variable based on them.
+            
+            e.g. the following example imports our **shell environment** ``HOME`` variable and sets the **TCL** 
+            ``HOME`` variable with it. The **TCL** ``HOME`` variable is used to set the **TCL variable** ``MY_PROGRAM_DIR`` 
+            (the software's installation directory). The **TCL** ``MY_PROGRAM_DIR`` variable is then used to add the program's 
+            ``bin`` directory to your **shell environment** ``PATH`` variable with the ``prepend-path`` directive.
+
+                .. code-block:: TCL
```
Comment (Carldkennedy) [inline referenceinfo/environment-modules/creating-custom-modulefiles.rst:111]:
```suggestion
            .. code-block:: TCL
```

Comment (Carldkennedy) [reply referenceinfo/environment-modules/creating-custom-modulefiles.rst:111]: Fixes code block

**Review by Carldkennedy — COMMENTED** [review]
LGTM. Added one suggestion to correct code-block

**Review by nikishe — APPROVED** [review]
Nice work

**Pull request merged**